cmd/drone-server/server.go

```diff
@@ -101,12 +101,13 @@ var flags = []cli.Flag{
 	cli.DurationFlag{
 		EnvVar: "DRONE_SESSION_EXPIRES",
 		Name:   "session-expires",
-		Usage:  "Set the session expiration time default 72h",
+		Usage:  "set the session expiration time default 72h",
```

Comment:
I think we can skip the `default 72h` because it's anyway part of the help output:

```
   --session-expires value            Set the session expiration time default 72h (default: 72h0m0s) [$DRONE_SESSION_EXPIRES]
```